prestosql/cdh5-base/Dockerfile

```diff
@@ -48,7 +48,6 @@ RUN wget -nv http://archive.cloudera.com/cdh5/one-click-install/redhat/6/x86_64/
 COPY files/startup.sh /root/startup.sh
 COPY files/supervisord.conf /etc/supervisord.conf
 COPY files/supervisord.d/bootstrap.conf /etc/supervisord.d/bootstrap.conf
-COPY files/change_timezone.sh /root/change_timezone.sh
```

Comment:
This is needed by WIP work on fixing timestamp semantics by @fiedukow. So tests do not use UTC which tends to hide bugs.
I have high hopes that work will be picked up by someone so please leave this.